go.mod

```diff
@@ -120,8 +120,12 @@ require (
 
 replace (
 	github.com/cosmos/cosmos-sdk => github.com/osmosis-labs/cosmos-sdk v0.43.0-rc3.0.20211119062100-2f81f59f3aa1
+	github.com/cosmos/go-bip39 => github.com/cosmos/go-bip39 v1.0.0
+	github.com/cosmos/iavl => github.com/cosmos/iavl v0.17.3
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+	github.com/tecbot/gorocksdb => github.com/cosmos/gorocksdb v0.0.0-20211202124722-2c356d6d98e4
```

Comment:
ditto for my comment re: bbolt. Do we have to upgrade this every patch version of TM/SDK/TM-db?

I'm more happy to do this if we decide to give more rocksdb support, but I want to be clear on why were getting multiple versions, and if we can do this replace in (say) our SDK fork, or a tm-db fork.

Reply:
It seems to me that they've resolved this issue in tm-db v0.6.6